chips/cc26xx/src/aon.rs

```diff
@@ -0,0 +1,42 @@
+//! Always On module
```

Comment:
What is the purpose of the always on module? It might be nice to add another comment here to clarify what it's in charge of

Reply:
Nice catch, it's easy to assume that something is obvious when you expose yourself to it very frequently!

The AON (Always On) stuff is basically peripherals and features which can not be turned of, and is always on (eg. RTC). I'll update the file and add some documentation :)